content/resources/registry/publishing/_index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: How to publish a recognizer
+image: images/Maybe-home-page3.jpg
+layout: resources
+---
+
+# Why publish your recognizer?
+Publishing your recognizer can be your way to contribute to open science and help the development of ecoacoustics as a tool for conservation. By sharing your work and making it publicly available you allow others in the scientific community to benefit from and build upon your research. We highly appreciate and value your contributions! 
+
+# How  to publish your recognizer
+We recommend using our [Recognizer Template in GitHub](https://github.com/ecoacoustics/recognizer-template) for publishing your recognizer. If you are familiar with GitHub you can proceed directly to the template. However, if you need more information on how to use it effectively, please continue reading this article.
+
+## What is GitHub and why is it great for this purpose?
+GitHub is a web-based service that utilises Git, which is a tool for version control. Git allows you to track changes, collaborate with others and manage different versions of your work. GitHub provides a platform for hosting and sharing Git repositories. You can read more about git and its uses from [GitHub documentation page](https://docs.github.com/en/get-started/using-git/about-git).
+
+### Version Control
+Version control is a system for recording changes to files over time, allowing you to track any modifications and revert back to previous versions. Version control is great for making collaboration with others seamless and manageable.
+
+### Zenodo - DOI generation
+Zenodo is an open-source project for generating DOIs that integrates well with GitHub. We recommend using Zenodo for generating a permanent DOI (Digital Object Identifier) for your recognizer. A DOI provides a unique and persistent identifier for your work, enabling others to cite it in their publications. Obtaining a DOI through Zenodo enhances the discoverability and visibility of yopur recognizer contributing to the impact and recoginition of your work within the scientific community.
+
+## How do I use the GitHub template to publish my recognizer?
```

Comment:
I think we need a section before this that explains what goes in the repository. Particularly since the template is designed to hold one or more recognizers.

- What goes in the repository?
- One or more recognizers, which should consist of:
    - code to run the recognizers
    - the code that makes the recognizers 
    - test and training data (or instructions on how to access said data)
    - and models or artifacts (or instructions on how to access said data)
- Some information in the readme that describes
    - what species are detected by the recognizer
    - the performance of the recognizer
    - license and attribution information

Reply:
And maybe TODO once Phil has done one, a link to some examples.